sampling.js

```diff
@@ -144,6 +146,19 @@ function sampleFromRegions(inRegions, numSamples, binSize) {
   return sampledRegions;
 }
 
+function sampleMore (inRegions) {
+  if (sampleMultiplier < sampleMultiplierLimit) {
+    sampleMultiplier += 1;
+    TARGET_BIN_SIZE = TARGET_BIN_SIZE + (TARGET_BIN_SIZE / 4) * sampleMultiplier;
```

Comment:
`TARGET_BIN_SIZE`  is uppercase because it's a constant. That's a naming convention. If we're going to modify it, it should be `targetBinSize`. However, we don't want to modify global variables like this. There could be another piece of code in an application that expects the sampling to use the default values, but they will be overwritten if this code is called. Instead of sharing a global, let's keep using the old function and update it to take a multiplier parameter:

```javascript
function sample(inRegions, multiplier) {
  ...
}
```

Reply:
Before the code review, I want a clarification from you, if we pass the `samplingMultiplier` into sample() to update the `TARGET_BIN_SIZE` and `NUM_SAMPLES`. How about the `expandRegion(region)`, because these two variables are not only used in `sample().`

Reply:
Sorry running behind again. Yeah I think we want to updated the other functions so that we pass down the sampling multiplier